coxeter/shapes/convex_spheropolygon.py

```diff
@@ -61,7 +61,7 @@ def reorder_verts(self, clockwise=False, ref_index=0, increasing_length=True):
         """Sort the vertices.
 
         For more information see
-        :meth:`~coxeter.shape_classes._polygon.Polygon.reorder_verts`.
+        :meth:`~coxeter.shapes._polygon.Polygon.reorder_verts`.
```

Comment:
This is the preferred alias, right? (This also might just be a messed up reference that was supposed to say `convex_polygon`)
```suggestion
        :meth:`~coxeter.shapes.Polygon.reorder_verts`.
```